api/swagger.yaml

```diff
@@ -2495,6 +2495,17 @@ definitions:
         example:
           - "server_x"
           - "server_y"
+      DriverOpts:
```

Comment:
I could add this ...

```
* `POST /containers/create` field `NetworkingConfig.EndpointsConfig.DriverOpts`,
  and `POST /networks/{id}/connect` field `EndpointsConfig.DriverOpts`, now
  support label `com.docker.network.endpoint.sysctls` for setting per-interface
  syscts. The value is a comma separated list of shortened sysctl assignments, the
  prefix `net.` and the interface name must be omitted. For example, 
  `net.ipv4.config.eth0.log_martians=1` must be shortened to
  `ipv4.config.log_martians=1`.
```

... but it might be a mistake to start documenting the use of labels as changes in the API? They're opaque values from the API's perspective.

The implementation isn't driver specific, so it is a bit weird that it's a driver-opt. But, that's just implementation - passing the DriverOpts to drivers isn't new.

WDYT?

Reply:
Oh, it is worth documenting the migration from top-level `--sysctl` to `DriverOpts` ... that is API behaviour, and I can mention that it'll stop happening in API version 1.47.